Xpresso/model-template.yaml

```diff
@@ -0,0 +1,43 @@
+defined_as: model.Xpresso
+args:
+    weights:
+        url: {{ model_weights_url }}
+        md5: {{ model_weights_md5 }}
+        #url: https://zenodo.org/record/4075690/files/humanMedian_trainepoch.11-0.426.h5
+        #md5: 9d00a3bc614da81655328b6e278569e2
+
+default_dataloader:
+    defined_as: kipoiseq.dataloaders.SeqIntervalDl
+
+    default_args:
+        alphabet_axis: 1
+        alphabet: "ACGT" # Order of letters in 1-hot encoding
+        use_strand: True
+        ignore_targets: False # if True, dont return any target variables
+
+info: # General information about the model
+    authors:
+        - name: Vikram Agarwal
+          github: vagarwal87
+          email: vagar@calicolabs.com
+    doc: A model to predict RNA expression levels from a genomic sequence
+    cite_as: https://doi.org/10.1016/j.celrep.2020.107663
+    trained_on: A random subset of ~17,000 protein-coding gene promoters
+    license: MIT
+
+dependencies:
+    conda: # install via conda
+      - python=3.5
+      - h5py
+      - numpy
+    pip:   # install via pip
+      - keras>=2.2.4
```

Comment:
I'd pin these two dependencies to their exact versions (used when tests successfully ran)

Reply:
Can you guys let me know please?
@Karollus @vagarwal87